CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Backup / Overview] Remove cancel button [#4657](https://github.com/vatesfr/xen-orchestra/issues/4657) (PR [#4688](https://github.com/vatesfr/xen-orchestra/pull/4688))
```

Comment:
@Rajaa-BARHTAOUI @pdonias @julien-f  When should we notify the users that the job cancel functionality was been removed because it not works.

Reply:
Maybe in a "Dropped features" section. Is it temporary, will it work in the future?

Reply:
The goal is for it to be temporary because I want to have working cancellation soon when I'm done with the rewrite.

IMHO, disabling the button with a tooltip explaning it's temporary disabled would be better.

Reply:
Copy, i'll disable it instead :+1:

Reply:
It's just an idea, do it if you agree with me 🙂 

@pdonias What do you think?

Reply:
Yes, if it's supposed to come back later, we can keep all the code for it, disable the button, add a tooltip, and add a comment that explains that and what changes need to be done once the feature is available again.